appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1517,6 +1517,10 @@ private static int upgradeTextBoxProperties(Map<String, JSONValue> componentProp
       // RequestFocus method was added
       srcCompVersion = 5;
     }
+    if (scrCompVersion < 6) {
```

Comment:
Compile failed. This is a typo that should be srcCompVersion.

Reply:
@SusanRatiLane just fixed that. Thanks for pointing this out!